runtime/gc_include/j9modron.h

```diff
@@ -145,6 +145,7 @@ typedef jvmtiIterationControl J9MODRON_REFERENCE_CHAIN_WALKER_CALLBACK(J9Object
 #define J9GC_REFERENCE_TYPE_CLASS_ARRAY_CLASS -12 /**< reference to an array class' component type class */
 #define J9GC_REFERENCE_TYPE_CLASS_NAME_STRING -13 /**< reference to a class' name string */
 #define J9GC_REFERENCE_TYPE_CALL_SITE -14 /**< call site reference to an object */
+#define J9GC_REFERENCE_TYPE_FCC_CLASS -15 /**< reference to a class in flattened class cache */
```

Comment:
I am sorry, I think `J9GC_REFERENCE_TYPE_CLASS_FCC` is better for consistency with `J9GC_REFERENCE_TYPE_CLASS_ARRAY_CLASS` and `J9GC_REFERENCE_TYPE_CALL_SITE`